client/src/components/forms/FormHOC/FormHOC.tsx

```diff
@@ -0,0 +1,22 @@
+import s from './FormHOC.module.less';
+import { Card, Form, Typography } from 'antd';
+import type { FormProps } from 'antd';
+import type { FC, PropsWithChildren, ReactNode } from 'react';
+
+interface Props extends Omit<FormProps, 'children' | 'title'>, PropsWithChildren {
+  title?: ReactNode;
+  formTitle?: FormProps['title'];
+}
+
+export const FormHOC: FC<Props> = ({ formTitle, title, children, ...rest }) => {
```

Comment:
What does `HOC` stand for?

Reply:
HigherOrderComponent - but thanks to your question, i noticed that naming it that way was an overkill - changed to `<FormWrapper>`